### DISCUSSION TITLE
Please clarify whether a single code field can ever include multiple codes

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/hospital-price-transparency#119] [Q&A] [answered]
We are seeing hospitals that are including multiple codes in a single field.  They are using semi-colon to separate the codes.
A specific example we found is including several revenue codes as follows: "118; 120; 128; 138; 148; 158"
I would guess they are trying to use this as an OR -- any of these Revenue Codes would get the same charges, but it is hard to tell. This doesn't seem to fit the specification.

However, the description of the code field doesn't explicitly say that each code field must have one and only one code -- it is a slightly ambiguous in it use of "code(s)":

> Any code(s) used by the hospital for purposes of billing or accounting for the item or service. See [additional csv placeholder notes](https://github.com/CMSgov/hospital-price-transparency/discussions/new?category=q-a#additional-csv-placeholder-notes) for implementation details.

It would be great if this description was tightened up so it is clear.  Two clarifications would help:

* one code per field
* AND not OR: when there are multiple codes (and code types) on the same row (in separate fields), the charges on that row only apply when ALL of the codes apply to the claim

Does this match what was intended?

**Comment by RidenourK** [comment, marked as answer]
@nmaylevc. Thank you for your suggestion that CMS clarify in future guidance whether a single code field can include multiple codes.

**Comment by angelina-abitino** [comment]
I agree that this needs to be clarified. The code set is one of the most important pieces of rate information. At a minimum, we need to know if the rates posted should apply to the combination of codes within a CSV row or if the rate can be applies to each code individually. 

Since there is so much flexibility with the code/type fields (unlimited number of code/type pairs allowed, can have multiple codes of the same type within a row), it makes it very difficult to aggregate the data. Should we expand a CSV row into one row per code/type pair, each row associated with the same rate (essentially duplicate all other data)? Or should the rate only be applied to the full combination of codes provided in the row?
